Alex and Happy Part 1/article.md

```diff
@@ -0,0 +1,817 @@
+# Parsing with Haskell (Part 1): Lexing with Alex
+
+<!--
+* [X] generic introduction
+-->
+
+This is the first part of the two parts _Parsing with Haskell_ series.
+Looking for the second part instead? You can find it [here](https://serokell.io/blog/parsing-with-happy).
+
+A critical part of the majority of the compilers is grammar.
```

Comment:
Are these two lines necessary? I don't believe they add to the introduction, but I might not understand their relevance here.